openpype/settings/entities/schemas/projects_schema/schemas/schema_templated_workfile_build.json

```diff
@@ -25,8 +25,15 @@
                     {
                         "key": "path",
                         "label": "Path to template",
-                        "type": "text",
-                        "object_type": "text"
+                        "type": "path",
+                        "multiplatform": false,
+                        "multipath": false
+                    },
+                    {
+                        "key": "keep_placeholder",
```

Comment:
This should be added only to nuke (the setting is there for maya too but does not affect the result).